Remotion/Data/DomainObjects.Validation/DomainObjectAttributesBasedValidationPropertyRuleReflector.cs

```diff
@@ -170,15 +170,22 @@ public IEnumerable<IPropertyValidator> GetRemovablePropertyValidators ()
             _validationMessageFactory);
       }
 
-      if (!_domainModelConstraintProvider.IsNullable(_implementationPropertyInformation)
-          && typeof(IEnumerable).IsAssignableFrom(_implementationProperty.PropertyType)
-          && !ReflectionUtility.IsObjectList(_implementationProperty.PropertyType)
-          && !ReflectionUtility.IsIObjectList(_implementationProperty.PropertyType))
+      if (!_domainModelConstraintProvider.IsNullable(_implementationPropertyInformation))
       {
-        yield return PropertyValidatorFactory.Create(
-            _implementationPropertyInformation,
-            parameters => new NotEmptyValidator(parameters.ValidationMessage),
-            _validationMessageFactory);
+        if (typeof(string).IsAssignableFrom(_implementationProperty.PropertyType))
```

Comment:
Same as with NotEmptyValidationAttribute:
```suggestion
        if (_implementationProperty.PropertyType == typeof(string))
```